StellarApp/Controllers/SolarDetailViewController.swift

```diff
@@ -1,30 +1,20 @@
-//
-//  SolarDetailViewController.swift
-//  StellarApp
-//
-//  Created by TingxinLi on 4/8/19.
-//  Copyright © 2019 TingxinLi. All rights reserved.
-//
-
 import UIKit
 
 class SolarDetailViewController: UIViewController {
 
+    let solarDetailView = SolarDetailView()
     override func viewDidLoad() {
         super.viewDidLoad()
+        view.backgroundColor = .blue
 
-        // Do any additional setup after loading the view.
-    }
-    
+        view.backgroundColor = #colorLiteral(red: 0.4620226622, green: 0.8382837176, blue: 1, alpha: 1)
```

Comment:
color literals are fine, but IMO it's better to extend UIColor to include these colors as it's more readable.